index.html

```diff
@@ -0,0 +1,286 @@
+            <h2 class="card__title">
+                Host yourself
+            </h2>
+
+            <p class="card__subtitle">
+                Wonder about your data?
+            </p>
+
+            <p>
+                Start by <a href="https://github.com/FreshRSS/FreshRSS/releases/latest">downloading the code</a>
+                and follow <a href="https://github.com/FreshRSS/FreshRSS#installation">the instructions to install it</a>
+                on your server.
+            </p>
+
+            <img alt="" src="images/server.svg" width="100">
+        </div>
+    </div>
+</div>
+
+<div class="wrapper-large edge-gap flow-small">
+    <h2 class="title">A powerful aggregator</h2>
+
+    <div class="cols">
+        <div class="card flow-small">
+            <h3 class="card__title">
+                Syndication
+            </h3>
+
+            <p>
+                Follow websites, podcasts and video channels in a single place.
+            </p>
+        </div>
+
+        <div class="card flow-small">
+            <h3 class="card__title">
+                Reader
+            </h3>
+
+            <p>
+                Read your articles directly in FreshRSS.
+            </p>
+        </div>
+
+        <div class="card flow-small">
+            <h3 class="card__title">
+                Search
+            </h3>
+
+            <p>
+                Search and save queries for quick access.
+            </p>
+        </div>
+    </div>
+
+    <div class="cols">
+        <div class="card flow-small">
+            <h3 class="card__title">
+                Web scraping
+            </h3>
+
+            <p>
+                Generate feeds by scraping external websites.
+            </p>
+        </div>
+
+        <div class="card flow-small">
+            <h3 class="card__title">
+                Feeds generation
+            </h3>
+
+            <p>
+                Generate new feeds based on your filters.
+            </p>
+        </div>
+
+        <div class="card flow-small">
+            <h3 class="card__title">
+                <abbr>OPML</abbr>
+            </h3>
+
+            <p>
+                Import and export your feeds with <abbr>OPML</abbr>.
+            </p>
+        </div>
+    </div>
+
+    <div class="cols">
+        <div class="card flow-small">
+            <h3 class="card__title">
+                WebSub
+            </h3>
+
+            <p>
+                Stay connected to your feeds in real-time.
+            </p>
+        </div>
+
+        <div class="card flow-small">
+            <h3 class="card__title">
+                Configurable
+            </h3>
+
+            <p>
+                Adapt to your needs thanks to a lot of options.
+            </p>
+        </div>
+
+        <div class="card flow-small">
+            <h3 class="card__title">
+                Mobile
+            </h3>
+
+            <p>
+                Follow your feeds with or without third-party application.
```

Comment:
```suggestion
                Follow your feeds with or without [third-party application](https://github.com/FreshRSS/FreshRSS/blob/edge/README.md#apis--native-apps).
```

Reply:
(applied a small edit to @Alkarex' suggestion)